drivers/soundwire/stream.c

```diff
@@ -281,6 +281,9 @@ static int sdw_program_port_params(struct sdw_master_runtime *m_rt)
 	/* Program transport & port parameters for Slave(s) */
 	list_for_each_entry(s_rt, &m_rt->slave_rt_list, m_rt_node) {
 		list_for_each_entry(p_rt, &s_rt->port_list, port_node) {
+			ret = sdw_slave_set_frequency(s_rt->slave);
+			if (ret < 0)
+				return ret;
```

Comment:
Commit message: it is a good practice if the "commit title" and the "commit message" alone can be understood, generally referring from the message to the title is not nice as one needs to connect the dots...

Message:
Update the peripheral frequency before setting the port params as the curr_dr_freq might have been changed since it's initialization.

Reply:
> Commit message: it is a good practice if the "commit title" and the "commit message" alone can be understood, generally referring from the message to the title is not nice as one needs to connect the dots...
> 
> Message: Update the peripheral frequency before setting the port params as the curr_dr_freq might have been changed since it's initialization.

Updated